### ISSUE TITLE
MTR 3.2.2 Track, stations, elevator, fails, to save In server 

### ISSUE THREAD

**Issue by HENSOLO-PSD** [Minecraft-Transit-Railway/Minecraft-Transit-Railway#912]
### Before reporting, I have confirmed that

- [X] This bug does not appear to be reported on [GitHub Issues](https://github.com/Minecraft-Transit-Railway/Minecraft-Transit-Railway/issues?q=is%3Aissue) before.
- [X] This bug still exists in the latest version of the Minecraft Transit Railway mod.
- [X] I have tested this in the official version of Minecraft Transit Railway, not a modified, custom, or unofficial build.

### Describe the Bug

Every time I simply place down a track. or name a station even placing an elevator. It works perfectly. until you turn off the server and you come back to the server later on and the stuff is gone. The rails is gone node Is invisible And the elevator is gone. and the stations are wipe. No names, no nothing. Not even a depot. Its like im starting from ground zero again. And is pretty sad because. everybody inside the server love this mod. It brings the life just traveling places and we can't do that because of the issue. I even tried fixing it using nemo transit mod and that didn't even work It just resort to the same issue. It used to work perfectly around. June until now it doesn't work.

### Reproduction Steps

1. Simply placed down some tracks and names stations.
2. Turns off the server.
3. Turn the server back on. None of the stuff that I placed down saved.

### In-game Log and Crash Report

[22Jul2024 00:04:11.515] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: devanohdevan was blown up by Wilden Chimera
[22Jul2024 00:04:11.522] [Server thread/INFO] [com.tiviacz.travelersbackpack.TravelersBackpack/]: Created new backpack backup file for devanohdevan with unique ID travelersbackpack.lapis_22.07.2024-00.04.11.dat
[22Jul2024 00:04:11.523] [Server thread/INFO] [com.tiviacz.travelersbackpack.TravelersBackpack/]: There's no space for backpack. Dropping backpack item at X: 6919 Y: 80.0 Z: -899
[22Jul2024 00:11:24.825] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: devanohdevan lost connection: Disconnected
[22Jul2024 00:11:24.826] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: devanohdevan left the game
[22Jul2024 00:12:39.661] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: KENDREX_ lost connection: Disconnected
[22Jul2024 00:12:39.662] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: KENDREX_ left the game
[22Jul2024 00:12:39.662] [Server thread/INFO] [de.geheimagentnr1.auto_restart.task.AutoRestartTask/]: Empty server timer started
[22Jul2024 00:16:37.355] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Stopping the server
[22Jul2024 00:16:37.406] [Server thread/INFO] [mtr.libraries.org.eclipse.jetty.server.AbstractConnector/]: Stopped ServerConnector@204c5aac{HTTP/1.1, (http/1.1)}{0.0.0.0:8888}
[22Jul2024 00:16:37.407] [Server thread/INFO] [mtr.libraries.org.eclipse.jetty.server.handler.ContextHandler/]: Stopped m.l.o.e.j.s.ServletContextHandler@9d6bd85{/,union:/C:/Users/kendr/Downloads/COBBLEMON%20ADVANCE%204.3.5/mods/MTR-forge-1.20.1-3.2.2-hotfix-2.jar%23421!/assets/mtr/website,STOPPED}
[22Jul2024 00:16:38.460] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Stopping server
[22Jul2024 00:16:38.461] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving players
[22Jul2024 00:16:38.461] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving worlds
[22Jul2024 00:16:38.572] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[New World]'/minecraft:overworld
[22Jul2024 00:16:38.665] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_end
[22Jul2024 00:16:38.666] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[New World]'/deeperdarker:otherside
[22Jul2024 00:16:38.667] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_nether
[22Jul2024 00:16:38.807] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage (COBBLEMON ADVANCE): All chunks are saved
[22Jul2024 00:16:38.807] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[22Jul2024 00:16:38.807] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage (otherside): All chunks are saved
[22Jul2024 00:16:38.808] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[22Jul2024 00:16:38.808] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage: All dimensions are saved
[22Jul2024 00:16:38.875] [Server thread/INFO] [de.geheimagentnr1.auto_restart.util.ServerRestarter/]: Saving restart status "STOP" to file
[22Jul2024 00:16:38.880] [Server thread/INFO] [dev.mikeretriever.cobblemonmikeskills.CobblemonMikeSkills/]: Mike Skills writing config files. . .
[22Jul2024 00:16:38.887] [Server thread/INFO] [Framework/]: Unloading server configs...
[22Jul2024 00:16:38.888] [Server thread/INFO] [Framework/]: Sending config unload event for mighty_mail.server.toml
[22Jul2024 00:16:38.937] [Server thread/INFO] [journeymap/]: Loading JourneyMap Forge Configs
[22Jul2024 00:16:39.139] [fr.denisd3d.mc2discord.shadow.reactor-http-nio-3/INFO] [fr.denisd3d.mc2discord.shadow.discord4j.core.shard.LocalShardCoordinator/]: [G:790fca49, S:0] Shard disconnected
[22Jul2024 00:16:39.140] [fr.denisd3d.mc2discord.shadow.reactor-http-nio-3/INFO] [fr.denisd3d.mc2discord.shadow.discord4j.core.shard.GatewayBootstrap/]: [G:790fca49, S:0] All shards disconnected
[22Jul2024 00:16:56.373] [pool-15-thread-1/WARN] [com.shutdowner.Shutdowner/]: Server shut down correctly, ending
[2024-07-21-1.log](https://github.com/user-attachments/files/16330185/2024-07-21-1.log)

### Mod Loader

Forge

### Minecraft Version

1.20.1

### Minecraft Transit Railway Version

3.2.2

### Operating System

Windows 11

### List of Mods or Modpack You Were Using

![image](https://github.com/user-attachments/assets/1948a253-570a-42c4-bfe5-f7a36aa16fa9)
![image](https://github.com/user-attachments/assets/b7c271e1-e3af-44cf-b573-b9c383ea158d)

**Comment by mingXsin** [comment]
Yeah, having the same issue too. The rails, elevator tracks are all missing even after saving with /save-all on terminal/shell

**Comment by nuclearxmoose** [comment]
Same issue,  A mod from the discord said it could be an issue with your hosting provider blocking the servers access to files,  I have later debunked this as the trains will not save in single player either.   It has to be a mod confliction or this version of forge is not working correctly>  I hope this info can help you solve this reoccurring issue.

**Comment by nuclearxmoose** [comment]
EVERYONE I FOUND THE ISSUE, you need to download the MTR file from their website NOT THE ONE LISTED ON CURSEFORGE, After this, i can reset my server and the rails stay on the server

**Comment by HENSOLO-PSD** [comment]
What do we have to download? Because it still shows curse forge and modrith

**Comment by HENSOLO-PSD** [comment]
Because the 4.0. version save. the rails completely. but not the 3.2.2

**Comment by nuclearxmoose** [comment]
this is the version https://modrinth.com/mod/minecraft-transit-railway/version/FORGE-4.0.0-beta-8+1.20.1

**Comment by nuclearxmoose** [comment]
yes the 4.0 works on hosted servers

**Comment by HENSOLO-PSD** [comment]
I think I'm gonna wait for beta 9 There is a extreme bad issue with. RAM and CPU usage going up.
